src/Npgsql/NpgsqlRawCopyStream.cs

```diff
@@ -61,7 +61,7 @@ internal NpgsqlRawCopyStream(NpgsqlConnector connector, string copyCommand)
             _connector.WriteQuery(copyCommand);
             _connector.Flush();
 
-            var msg = _connector.ReadMessage();
+            var msg = _connector.ReadMessageWithoutCancellation();
```

Comment:
We should do whatever we decide in NpgsqlBinaryExporter

Reply:
`RawCopyStream` has a different mechanism of the cancellation (there is a `CopyFail` written in the same connection). 
Also, here we're only starting the copy operation. I'm not sure it can timeout by itself (without network issues), and it is even OK to cancel it.

Reply:
RawCopyStream can do either import, or export - just like NpgsqlBinaryImporter/NpgsqlBinaryExporter (it's just more low-level). So when exporting, the same thing we do for NpgsqlBinaryExporter makes sense here.

However, I agree that cancelling during copy startup is probably useless. Let's leave it as-is.